OpenRA.Mods.RA/Traits/Infiltration/InfiltrateForCash.cs

```diff
@@ -17,9 +18,15 @@ namespace OpenRA.Mods.RA.Traits
 	[Desc("This structure can be infiltrated causing funds to be stolen.")]
 	class InfiltrateForCashInfo : ITraitInfo
 	{
-		public readonly int Percentage = 50;
-		public readonly int Minimum = 500;
-		public readonly string SoundToVictim = "credit1.aud";
+		[Desc("Percentage of the victim's resources that will be stolen.")]
+		public readonly int Percentage = 100;
+
+		[Desc("Amount of guaranteed funds to claim when the victim does not have enough resources.",
+			"When negative, the production price of the infiltrating actor will be used instead.")]
+		public readonly int Minimum = -1;
+
+		[Desc("Sound the victim will hear when they get robbed.")]
+		public readonly string SoundToVictim = null;
```

Comment:
Yeah, I was wondering why it referred directly to a sound file. Will have a look at that.